Gravatar.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
```

Comment:
tady chybí `declare(strict_types=1)`:

```suggestion
declare(strict_types=1);
```